shell.nix

```diff
@@ -1,53 +1,58 @@
-{ channel ? "nightly" }:
+{ nospdk ? false }:
 let
-  nixpkgs = (import ./nix/lib/nixPackages.nix) { };
-  pkgs = import nixpkgs {
-    config = { };
-    overlays = [ (import ./nix/mayastor-overlay.nix) ];
+  sources = import ./nix/sources.nix;
+  pkgs = import sources.nixpkgs {
+    overlays = [
+      (_: _: { inherit sources; })
+      (import ./nix/mayastor-overlay.nix)
+    ];
   };
 in
 with pkgs;
 let
-  rustChannel = import ./nix/lib/rust.nix {
-    inherit fetchFromGitHub;
-    inherit pkgs;
-  };
-  libspdk = pkgs.libspdk.override { enableDebug = true; };
+  moth = "You have requested environment without SPDK, you should provide it!";
+  channel = import ./nix/lib/rust.nix { inherit sources; };
 in
 mkShell {
+
   # fortify does not work with -O0 which is used by spdk when --enable-debug
   hardeningDisable = [ "fortify" ];
-
   buildInputs = [
-    figlet
+    #    nats-server
```

Comment:
nats-server is needed by the tests. Was it removed by accident?

Reply:
Oh yes -- restored, tnx!